environments/stg/stg.tfvars

```diff
@@ -25,6 +25,13 @@ cft_apps_ag_ip_address = "10.10.161.123"
 cft_apps_cluster_ips   = ["10.10.143.250", "10.10.159.250"]
 
 frontends = [
+  {
+    name           = "plum"
+    custom_domain  = "plum.aat.platform.hmcts.net"
```

Comment:
this dns record needs to be in place first

Reply:
https://github.com/hmcts/azure-public-dns/pull/926